node/status_node.go

```diff
@@ -176,7 +177,15 @@ func (n *StatusNode) setupRPCClient() (err error) {
 	return
 }
 
+func (n *StatusNode) discoveryEnabled() bool {
+	return n.config != nil && n.config.NoDiscovery && n.config.ClusterConfig != nil
```

Comment:
`!n.config.NoDiscovery`? (missing `!` in code)

Reply:
right, thanks